src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -315,7 +318,13 @@ void AbstractWriteLog<I>::log_perf() {
 template <typename I>
 void AbstractWriteLog<I>::periodic_stats() {
   std::lock_guard locker(m_lock);
-  ldout(m_image_ctx.cct, 1) << "STATS: m_log_entries=" << m_log_entries.size()
+  m_cache_state->bytes_allocated = m_bytes_allocated;
+  m_cache_state->bytes_available = m_bytes_allocated_cap - m_bytes_allocated;
+  m_cache_state->bytes_dirty = m_bytes_dirty;
+  m_cache_state->bytes_read = m_perfcounter->get(l_librbd_pwl_rd_bytes);
```

Comment:
In addition to the number of hit/missed read bytes, I would include the number of hit/missed read requests as well.